src/components/subscribe/subscribe.scss

```diff
@@ -43,7 +45,10 @@
   inline-size: 100%;
   inset-block-start: 0;
   inset-inline-start: 0;
-  position: absolute;
+
+  .c-subscribe:not(.is-destroyed) & {
+    position: absolute; // 2
+  }
```

Comment:
Could this be simplified to `&:not(:only-child)` so it doesn't require the state class?

Reply:
@tylersticka I _resolved_ this conversation prematurely. Sorry!